From 31c400d67c83ad8c9bb222f71c0250dbf600df64 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?S=C3=A9rgio=20M=2E=20Basto?= <sergio@serjux.com>
Date: Wed, 30 Jan 2019 18:19:43 +0000
Subject: [PATCH] 04_fix-abs-gcc-7.patch from Debian

Description: fix compilation in GCC 7
 change some variables to int instead of unsigned so that calls to abs are no
 longer ambiguous under GCC7
Author: Ken Moffat
Origin: https://sourceforge.net/p/ufraw/mailman/message/35843737/
Bug-Debian: https://bugs.debian.org/853692
---
 dcraw.cc | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/dcraw.cc b/dcraw.cc
index e5fba53..9014b73 100644
--- a/dcraw.cc
+++ b/dcraw.cc
@@ -8866,8 +8866,8 @@ void CLASS identify()
       "Nikon", "Nokia", "Olympus", "Ricoh", "Pentax", "Phase One",
       "Samsung", "Sigma", "Sinar", "Sony" };
   char head[32], *cp;
-  unsigned fsize, i, c;
-  int hlen, flen, zero_fsize=1;
+  unsigned fsize;
+  int hlen, flen, zero_fsize=1, i, c;
   struct jhead jh;
 
   tiff_flip = flip = filters = UINT_MAX;	/* unknown */
